Auctionator.lua

```diff
@@ -4659,7 +4655,6 @@ function Atr_BuildGlobalHistoryList()
     Atr_BuildActiveAuctions()
   end
 
-  local info, IDstring
```

Comment:
info and IDstring are stored locally inside of the for loop, doesn't require this